changelog/fragments/ansible-path-issue-fix.yaml

```diff
@@ -0,0 +1,9 @@
+entries:
+  - description: >
+      For Ansible operators, API endpoint has path component in it. If it has path then current flow is ignoring the path component.
```

Comment:
```suggestion
      For Ansible operators, if an API endpoint has path component in it then the ansible-operator binary will terminate early with an error, due to a bug in the proxy that would cause the operator to target the wrong cluster.
```

Reply:
Modified